config/routes.rb

```diff
@@ -1,8 +1,12 @@
 Rails.application.routes.draw do
-  root to: "submissions#index"
+  root to: "posts#index"
 
   devise_for :users, controllers: { registrations: 'registrations' }
   resources :users, only: :show
 
-  resources :submissions
+  resources :posts do
+    resources :comments, only: [:create, :show] do
```

Comment:
I didn't see a `show` action on the controller nor an associated view. Intentional?

Reply:
I was thinking about using a comments#show when I was brainstorming ways to handle the error messages, but ended up deciding not to.